app/assets/javascripts/application.js

```diff
@@ -40,8 +40,13 @@
 //= require controllers/ops/log_collection_form_controller
 //= require controllers/ops/tenant_form_controller
 //= require controllers/ops/tenant_quota_form_controller
+//= require controllers/container_dashboard/util/util-module
```

Comment:
> what is util-module?
> the name is a bit vague, and it's also under util folder, and under it there's also dashboard-utils...

@abonas Yeah, I guess its a little misleading. util-module is just a one line file defining the module.
 containers_dashboard needs a few resuable assets. dashboard-utils and ```dashboardUtilsFactory``` within it should probably be something like ```dashboardObjFactory```